set.mm

```diff
@@ -26079,14 +26079,29 @@ the definition of class equality ( ~ df-cleq ).  Its forward implication
   $}
 
   ${
-    $d A x y z $.  $d V y z $.
+    $d A x z $.  $d A y z $.
+    $( The expression ` E. x x = A ` does not depend on a particular choice of
+       the set variable.  Extracted from the proof of ~ elisset .  This version
+       is proven from ~ df-clab and ~ ax-8 .  Use it in contexts where you
+       still need not ~ df-cleq or ~ ax-ext elsewhere in your proof.  A shorter
+       version ~ cbvexeqsetv is not using ~ df-clab , and preferred once
+       ~ df-cleq has become ubiquitous.  (Contributed by BJ, 6-Aug-2025.) $)
```

Comment:
Probably more faithful to history would be:
> ...  (Contributed by BJ, 29-Apr-2019.) Extract from the proof of ~ elisset .  (Proof shortened by WL, 6-Aug-2025.) $)


As for labels, since both statements are the same, they sould be xxx and xxxALT. I'm not attached to which is the ALT. A good choice for "xxx" would be `issetjust`: as I explain in the comment of https://us.metamath.org/mpeuni/bj-denotes.html, this is the justification theorem for the definition of "is a set". We will not add that definition explicitly, but that label is probably a good mnemonic. (The cbv* theorems are typically for the general cases with formula variables; sorry for having advised it at https://github.com/metamath/set.mm/pull/4959#issuecomment-3150011232).